### DISCUSSION TITLE
Keyball44について、1_右手の特定キーが反応しない、2_剥がれてしまった

### DISCUSSION THREAD

**Discussion by sbsyou** [Yowkees/keyball#478] [Q&A] [answered]
Keyball44について、2件質問がございます。

1_青枠で囲ったキーが反応しません。キーソケットを差し替えましたが、変わらず反応しません。対応するダイオードなどやチェックする項目があれば、教えていただきたいです。

2_赤枠で囲った箇所が剥がれてしまったようです。こちらのキーを復活させるために、繋ぐ場所があれば教えていただきたいです。

よろしくお願いします。
![IMG_3382](https://github.com/Yowkees/keyball/assets/158572235/6968bc34-cba6-4ffa-85d4-e695cbe42912)

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

1つ目の青枠の反応が無いキーにつきましては画像の青い丸で囲ったダイオードが対応するダイオードになります。
キーソケットのはんだ付けも併せてご確認ください。

2つ目の赤枠の剥がれにつきましては、赤点で示した場所をエナメル線かゴム被覆線でつないでください。

![301873826-6968bc34-cba6-4ffa-85d4-e695cbe42912](https://github.com/Yowkees/keyball/assets/143698349/94f869b8-542c-467e-83e2-87c6a66af717)

**Reply by sbsyou** [reply, marked as answer]
ご返信いただきありがとうございます。パーツ揃い次第、対応します。
